spec/models/book_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
How would you guys feel about removing this setting from hound? @samnang @ajkamel